docs/how-to-guides/upgrade-bootstrap-kubelet.md

```diff
@@ -0,0 +1,44 @@
+# How to upgrade bootstrap Kubelet
+
+## Introduction
+
+Bootstrap Kubelet is the Kubelet running on the node as a systemd service. Unlike most components on the cluster, lokoctl cannot update it. So there are some manual steps involved in updating the bootstrap Kubelet.
+
+## Steps
+
+Apply the following steps to all the nodes one at a time.
```

Comment:
One at a time is not strictly necessary. I guess it's fine for the time being, but in the long run, we should explain, how to do a rollout without downtime.

Reply:
I am being conservative with the amount of changes done simultaneously to reduce downtime.